seatunnel-api/src/main/java/org/apache/seatunnel/api/sink/DataSaveMode.java

```diff
@@ -32,5 +32,8 @@ public enum DataSaveMode {
     CUSTOM_PROCESSING,
 
     // When there exist data, an error will be reported
-    ERROR_WHEN_DATA_EXISTS
+    ERROR_WHEN_DATA_EXISTS,
+
+    // Ignore
+    IGNORE
```

Comment:
DataSaveMode doesn't need `IGNORE`. The behavior of `IGNORE` same as `APPEND_DATA`.

Reply:
ok, thanks. I want to try again